app/graphql/resolvers/corpus_slate_resolvers.py

```diff
@@ -31,8 +32,18 @@ async def resolve_new_tab_slate(
             description="The geographic region, for example 'FR' or 'IT', or null if unavailable. This is currently not"
                         " used, but will be used in the future to decide which scheduled surface to return when we"
                         " serve multiple markets with the same language.")],
+        enable_ranking_by_region: Annotated[Optional[bool], strawberry.argument(
+            description="Experimental parameter that when enabled, adjusts the ranking of items based on the region."
+                        " By default this is off. Firefox Desktop will enable it by setting the pref "
+                        " browser.newtabpage.activity-stream.discoverystream.pocket-feed-parameters to \"&enableRankingByRegion=1\""
+                        " for the treatment branch of a Nimbus experiment."
+        )] = False,
```

Comment:
The GraphQL library we use automatically converts `enable_ranking_by_region` (conventional in Python) to `enableRankingByRegion` (conventional in GraphQL).